chainList.json

```diff
@@ -357,4 +375,4 @@
       "chain": "sepolia-dev-0"
     }
   }
-]
```

Comment:
This edit is causing the codegen ci test to fail. Please run `just codegen` again and push the change

Reply:
Fixed. This change was caused by this line in .editorconfig, it seems to be inconsistent with the CI pipeline: https://github.com/ethereum-optimism/superchain-registry/blob/ea7e768a02a6f88a38a1d39d27b9be6c212f4b6e/.editorconfig#L9